cluster.yml

```diff
@@ -30,15 +30,24 @@
     - { role: docker, tags: docker }
     - { role: rkt, tags: rkt, when: "'rkt' in [ etcd_deployment_type, kubelet_deployment_type ]" }
 
-- hosts: etcd:!k8s-cluster
+- hosts: all
+  any_errors_fatal: true
+  roles:
+    - { role: vault, tags: vault, vault_bootstrap: true, when: "cert_management == 'vault'" }
+
+- hosts: etcd:k8s-cluster
```

Comment:
Actually, in testing I realized that if your ``etcd`` nodes are included in your ``k8s-cluster``, then using ``etcd:!k8s-cluster`` will skip all nodes. I'll have to dig into the etcd role more after this, to understand why that's needed.